lib/rbnacl/signatures/ed25519/signing_key.rb

```diff
@@ -81,6 +81,20 @@ def sign(message)
           buffer[0, signature_bytes]
         end
 
+        # Sign a message using this key
+        #
+        # @param message [String] Message to be signed by this key
+        #
+        # @return [String] Signature and the message as bytes
+        def sign_full(message)
```

Comment:
Since the provided functionality is attached signatures, I think a name like `sign_attached` (and and correspondingly `verify_attached`) would make more sense.